modules/jaeger-config-storage.adoc

```diff
@@ -39,70 +28,58 @@ If you already have installed Elasticsearch as part of OpenShift logging, the Ja
 Memory storage is only appropriate for development, testing, demonstrations, and proof of concept environments as the data does not persist if the pod is shut down. For production environments Jaeger supports Elasticsearch for persistent storage.
 |`memory`
 
-|===
-
-
-.Elasticsearch configuration parameters
-[options="header"]
-[cols="l, a, a, a"]
-|===
-|Parameter |Description |Values |Default value
-
-4+|General Elasticsearch configuration settings
-|elasticsearch:
-  server-urls:
-|URL of the Elasticsearch instance.
-|The fully-qualified domain name of the Elasticsearch server.  If you have specified `spec:storage:type`= `elasticsearch` but have not specified a value for `server-urls` parameter, the Jaeger Operator will use the Elasticsearch Operator to create an Elasticsearch cluster using the configuration in the `spec:storage:elasticsearch` section of the custom resource file.
-|`http://elasticsearch.<namespace>.svc:9200`
-
-|es:
-  max-num-spans:
-|The maximum number of spans to fetch at a time, per query, in Elasticsearch.
-|
-|10000
-
-|es:
-  max-span-age:
-|The maximum lookback for spans in Elasticsearch.
-|
-|72h0m0s
-
-|elasticsearch:
+|storage:
   secretname:
 |Name of the secret, for example `jaeger-secret`.
 |
 |N/A
 
-|es:
-  sniffer:
-|The sniffer configuration for Elasticsearch.  The client uses the sniffing process to find all nodes automatically. Disabled by default.
+4+|Elasticsearch index cleaner configuration options
```

Comment:
I think this should be a separate table. I think if the table needs explanation halfway through, they're different enough to be in separate tables. Also, it's better for accessibility. But I also think two tables with three rows is more consumable than one with six.